Composer/packages/lib/shared/src/types/sdk.ts

```diff
@@ -111,7 +113,7 @@ export interface AttachmentInput extends Partial<InputDialog> {
 export interface ChoiceInput extends Partial<InputDialog> {
   /** The output format. */
   outputFormat?: 'value' | 'index';
-  choices?: IChoice[];
+  choices?: IChoice;
```

Comment:
Just a thought: What if we had an `AnyOf` type alias for this? I think it would make this easier to connect with the schema.